src/components/nodes/components/NodesTableHead.vue

```diff
@@ -37,22 +55,27 @@ export default {
     },
     label: {
       type: String
-    }
+    },
+    indeterminate: {
+      type: Boolean
+    },
   },
-  setup() {
+  setup(props, { emit }) {
```

Comment:
Define `emits: ['update:modelValue'],` above. You must see a warning in the console without it.

Reply:
I don't see the change. The `emits` property is not defined.

See example: https://github.com/Adamant-im/adamant-im/blob/c6c73452bbb1b8fbb0f907c611d5672442ad871a/src/components/ShareURIDialog.vue#L57